src/main/java/com/hackerrank/weather/controller/WeatherApiRestController.java

```diff
@@ -54,28 +55,40 @@ public List<Weather> weather(@RequestParam("date") String date) {
     }
 
     @GetMapping("/city/{cityName}")
-    public ArrayList<Double> getWeatherTemperatureByCityName(@PathVariable(name = "cityName") String cityName){
+    public ArrayList<Double> getWeatherTemperatureByCityName(@PathVariable(name = "cityName") String cityName) {
```

Comment:
Please, Add @ApiOperation with all properties that make sense for that endpoint.

Also, Use List<Double> it's always better to go with an interface instead of implementation. See the dependency inversion principle - [Link](https://stackify.com/dependency-inversion-principle/ )